src/lib/pipeline.js

```diff
@@ -312,24 +295,26 @@ class Pipeline extends Builder {
             pipeline.handleEvent(event);
           } else {
             monitor.hit('rejected');
-            pipeline.log(
+            errorLog(
               `Invalid message: ${validator.errorsText(validate.errors)}`,
               'warn'
             );
           }
         },
-        reject: function(reason) {
+        reject: reason => {
           monitor.hit('rejected');
-          pipeline.log(reason, 'warn');
+          errorLog(reason, 'warn');
         },
-        status: function(err, msg) {
+        status: (err, msg) => {
           if (err) {
             console.error(err);
           }
           console.log(input.name + ': ' + msg);
           monitor.status = msg;
         },
-        log: pipeline.log,
+        log: (err, severity) => {
```

Comment:
Is there any reason to do this instead of `log: errorLog` ?

Reply:
No reason to not write it like that.